tests/erp/test_commands.py

```diff
@@ -356,3 +355,59 @@ def test_max_results(self, mocker):
         )
         call_command("outscraper_acquisition", query="restaurant, Lyon", activity="Restaurant", max_results=30)
         assert mock.call_count == 2
+
+
+class TestOutscraperCleaning:
+    initial_outscraper_response = [
+        [
+            {
+                "query": "restaurant, Lyon",
+                "name": "Le lard",
+                "place_id": "ChIJzZhX5juz9UcR74W_abcd",
+                "google_id": "0x47f5b33be65798cd:0x8ac44e3b5cbf85ef",
+                "full_address": "140 Rue Trouvier, 69006 Lyon, France",
+                "street": "173 Rue Trouvier",
+                "postal_code": "69006",
+                "country_code": "FR",
+                "country": "France",
+                "city": "Lyon",
+                "latitude": 45.767984399999996,
+                "longitude": 4.8563522,
+                "site": "https://www.lelard.com/?utm_source=google+",
+                "phone": "+33 4 72 14 12 14",
+                "type": "Restaurant",
+                "description": "Restaurant gras.",
+                "category": "restaurants",
+                "subtypes": "Restaurant, Restaurant français, Restaurant grastronomique",
+                "cid": "9999203089535436271",
+                "business_status": "CLOSED_PERMANENTLY",
+                "about": {
+                    "Accessibilité": {
+                        "Entrée accessible en fauteuil roulant": True,
+                        "Toilettes accessibles en fauteuil roulant": True,
+                    },
+                },
+            }
+        ]
+    ]
+
+    @pytest.mark.django_db
+    def test_initial(self, mocker):
+        ErpFactory(nom="Le lard", commune="Lyon", voie="Rue Trouvier", numero=173)
+        mocker.patch("outscraper.ApiClient.google_maps_search", return_value=self.initial_outscraper_response)
+
+        call_command("outscraper_clean_closed", nb_days=0, write=False)
+        assert Erp.objects.filter(nom="Le lard").count() == 1, "should have deleted it"
```

Comment:
We don't want to delete in this case (right ?) so the assert message should be the other way around I believe ("should not have deleted the object")